pcdsdevices/fms.py

```diff
@@ -0,0 +1,259 @@
+"""
+CXI:SETRA:01:PERIODS_RBV 60 monitor
+CXI:SETRA:01:DELAY_TIME_RBV 60 monitor
+CXI:SETRA:01:SAMPLE_TIME_RBV 60 monitor
+CXI:SETRA:01:HOLD_TIME_RBV 60 monitor
+CXI:SETRA:01:SAMPLE_MODE_RBV 60 monitor
+CXI:SETRA:01:MASS_MODE_RBV 60 monitor
+CXI:SETRA:01:RECIPE_MODE_RBV 60 monitor
+CXI:SETRA:01:RECORD_REQ 60 monitor
+CXI:SETRA:01:RH 60 monitor
+CXI:SETRA:01:BP 60 monitor
+CXI:SETRA:01:TEMP 60 monitor
+CXI:SETRA:01:DATE 60 monitor
+CXI:SETRA:01:TIME 60 monitor
+CXI:SETRA:01:TVOC 60 monitor
+CXI:SETRA:01:CO2 60 monitor
+CXI:SETRA:01:FLOW_RATE 60 monitor
+CXI:SETRA:01:SAMPLE_DURATION 60 monitor
+CXI:SETRA:01:CH0_SIZE 60 monitor
+CXI:SETRA:01:CH0_MASS 60 monitor
+CXI:SETRA:01:CH1_SIZE 60 monitor
+CXI:SETRA:01:CH1_MASS 60 monitor
+CXI:SETRA:01:CH2_SIZE 60 monitor
+CXI:SETRA:01:CH2_MASS 60 monitor
+CXI:SETRA:01:CH3_SIZE 60 monitor
+CXI:SETRA:01:CH3_MASS 60 monitor
+CXI:SETRA:01:CH4_SIZE 60 monitor
+CXI:SETRA:01:CH4_MASS 60 monitor
+CXI:SETRA:01:CH5_SIZE 60 monitor
+CXI:SETRA:01:CH5_MASS 60 monitor
+"""
+
+
+from ophyd import Component as Cpt, Device, EpicsSignalRO
+from ophyd import FormattedComponent as FCpt, EpicsSignalRO, Device
+
+class Setra5000(Device):
+    Timestamp = Cpt(EpicsSignalRO, ':TIME', kind='hinted')
+    Date = Cpt(EpicsSignalRO, ':DATE', kind='hinted')
+
+    SampleState = Cpt(EpicsSignalRO, ':SAMP_STATE', kind='hinted')
+    SampleDuration = Cpt(EpicsSignalRO, ':SAMPLE_TIME_RBV', kind='hinted')
```

Comment:
Are these NamedLikeThis for the screens or something like that?
Usually we do e.g. `sample_duration` for these python attributes. But maybe there's a special reason for it? I'm interested because maybe we want to change how typhos displays these names...

Reply:
I don't think there is a particular reason for it. I changed the names in the fms.py file to match the standard you are referring to (sample_duration). What would need to be changed in typhos/what is typhos for?

Reply:
Typhos automatically generates a screen based on the text here and doesn't need any specific changes.